src/utils/experiments/checkUserBucket.ts

```diff
@@ -0,0 +1,26 @@
+import { getWikiaBeaconId } from '@fandom/context';
+import getValueFromQuery from 'utils/getValuefromQuery';
+
+export const DEFAULT_BUCKET_INDEX = 0;
+export const ALLOWED_BUCKET_CHAR = /[0-9a-zA-Z_-]/;
+
+export function getUserBucket(index = DEFAULT_BUCKET_INDEX): string {
+	const forcedBucket = getValueFromQuery('jw_force_experiment_bucket');
```

Comment:
Is this query param structured as `? jw_force_experiment_bucket=someBucket` ?

Reply:
Yup, thats correct, almost the same usage as SS/Pathfinder